PWE.html

```diff
@@ -30,61 +32,118 @@
+                
+            <p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally or in the context of the group. Participants should immediately report an incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.</p>
+                
+            <p>Everyone involved in the resolution process is expected to:</p>
+            
+            <ul>
+                <li>Show respect for others</li>
+                <li>Work together to resolve the complaint</li>
+                <li>Maintain confidentiality</li>
+            </ul>
+                
+            <p>A disciplinary investigation can be started where one of the following apply:</p>
+                
+            <ul>
+                <li>It’s not appropriate to begin with mediation</li> 
+                <li>All other mediation or resolution attempts have not worked</li>
+            </ul>
+        </section>
+        
+<h2> Dispute Resolution Guidance</h2>
+
+<p>The goal of dispute resolution is that individual participants in W3C can:</p>
+<ul><li>get help and support to safely resolve issues that negatively affect them</li>
+<li>correct their behaviors that negatively affect other participants in W3C</li>
+        <li>continue to safely work and contribute to the W3C after an issue has been raised</li>
+</ul>
+<p>The Code of Ethics and Professional Conduct is the standard that all W3C participants are required to follow and is used as the basis for any resolution and disciplinary actions. In exceptional circumstances, individuals may have their W3C participation suspended or withdrawn as a result of a complaint or investigation.</p>
+
+<p>All situations are different and W3C participants are encouraged to use whatever method of dispute resolution they feel comfortable with.
+</p>
+<p>Individuals are encouraged to try and resolve issues themselves in the first instance. However, if individuals do not feel comfortable or safe doing so, they can get help through the ombudsperson, facilitators and/or mediation. 
+</p>
+<p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally (for example, threats of violence, assault or sexual assault). Individuals should immediately report the incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.
+</p>
+<p>Everyone involved in the dispute resolution process is expected to:</p>
+<ul>
+     <li> show respect for others</li>
+    <li> work together to resolve the complaint</li>
+<li> maintain confidentiality</li>
+</ul>
+
+        <p>A disciplinary investigation can be started where one of the following apply:</p>
+<ul><li>it’s not appropriate to begin with mediation </li>
+<li> all other mediation or resolution attempts have not worked</li>
+</ul>

```

Comment:
The above 6 paragraphs are repetitions of what the Introduction already says, and should not be deleted to avoid the unnecessary repetition.